app/scripts/extract_scheduled_gtfs.py

```diff
@@ -146,70 +147,109 @@ def ingest_transit_stations(
             station_id=row["stop_id"],
             station_name=row["stop_name"],
             location=Point(row["stop_lat"], row["stop_lon"], srid=4326),
-            mode=transit_mode,
+            mode=assign_mode(row, url),
         )
         print(
             f'Observation created: {row["city"]}, {row["stop_id"]}, '
             f'{row["stop_name"]}, Point({row["stop_lat"]},{row["stop_lon"]})'
             f", mode {obs.mode}"
         )
-        print(obs)
         try:
```

Comment:
Good for this stage, but let's keep in mind for later to get more specific try-excepts